src/js/redux/reducers/search/pscReducer.js

```diff
@@ -100,6 +100,12 @@ export const pscReducer = (state = initialState, action) => {
                 counts: new List(action.payload)
             };
         }
+        case 'CLEAR_SEARCH_FILTER_ALL': {
```

Comment:
When does this get used?

Reply:
This happens when you hit "reset search" on the search sidebar.